src/java/org/apache/cassandra/locator/EverywhereStrategy.java

```diff
@@ -101,4 +101,14 @@ public void maybeWarnOnOptions()
     {
         return Collections.emptyList();
     }
+
+    /**
+     * @return <code>false</code> because the data is not partitioned across the ring.
+     * See APOLLO-589 for details about why this was introduced.
```

Comment:
I would remove/change this comment.  This code is OSS and someone outside DataStax can not see the internal JIRA...

Reply:
sure, changed